packages/client/src/views/SysAdmin/Team/user/UserList.tsx

```diff
@@ -466,25 +466,39 @@ function UserListComponent(props: IProps) {
   }
 
   function getStatusMenuType(
+    userDetails: IUserDetails | null,
+    locationId: string,
     user: GQLUser,
     index: number,
     status?: string,
     underInvestigation?: boolean
   ) {
+    const showMenuIcon =
```

Comment:
I think you can make this a little simpler by doing

```suggestion
    const canEditUserDetails = 
      userDetails && ['NATIONAL_SYSTEM_ADMIN', 'LOCAL_SYSTEM_ADMIN'].includes(userDetails.role) &&
      userDetails.primaryOffice?.id === locationId
```

Reply:
fixed it!